### PR TITLE
Feature/adjust dal

### PR DESCRIPTION
**Pull request by Gh0stF1ghter**

### CODE REVIEW

**Review by MaliyOna — APPROVED** [review]

File: DAL/Repositories/Implementations/ClientRepository.cs [hunk 1]
```diff
@@ -15,13 +15,12 @@ public async Task<IEnumerable<Client>> GetRangeAsync(int page, int pageSize, Can
         return await context.Clients
             .Skip(rowsToSkip)
             .Take(pageSize)
-            .Include(c => c.Vehicle).AsNoTracking()
+            .Include(c => c.VehicleClientHistory)
```
Comment (5Ka-me) [inline DAL/Repositories/Implementations/ClientRepository.cs:18]: Why did you remove _AsNoTracking()_?

**Review by 5Ka-me — APPROVED** [review]
